src/pages/teamMembers.js

```diff
@@ -0,0 +1,245 @@
+    phonenNo: "9898989898",
+    emailid: "pqrst@gmail.com"
+  },
+  {
+    name: "5",
+    designation: "Co-Founder",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/abcdef",
+    linkedin: "linkedin.com/in/abcdef",
+    phonenNo: "9898989898",
+    emailid: "abcdef@gmail.com"
+  },
+  {
+    name: "6",
+    designation: "President",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/pqrst",
+    linkedin: "linkedin.com/in/pqrst",
+    phonenNo: "9898989898",
+    emailid: "pqrst@gmail.com"
+  },
+  {
+    name: "7",
+    designation: "Member",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/abcdef",
+    linkedin: "linkedin.com/in/abcdef",
+    phonenNo: "9898989898",
+    emailid: "abcdef@gmail.com"
+  },
+  {
+    name: "8",
+    designation: "Founder",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/pqrst",
+    linkedin: "linkedin.com/in/pqrst",
+    phonenNo: "9898989898",
+    emailid: "pqrst@gmail.com"
+  },
+  {
+    name: "9",
+    designation: "Founder",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/abcdef",
+    linkedin: "linkedin.com/in/abcdef",
+    phonenNo: "9898989898",
+    emailid: "abcdef@gmail.com"
+  },
+  {
+    name: "10",
+    designation: "Founder",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/pqrst",
+    linkedin: "linkedin.com/in/pqrst",
+    phonenNo: "9898989898",
+    emailid: "pqrst@gmail.com"
+  },
+  {
+    name: "11",
+    designation: "Co-Founder",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/abcdef",
+    linkedin: "linkedin.com/in/abcdef",
+    phonenNo: "9898989898",
+    emailid: "abcdef@gmail.com"
+  },
+  {
+    name: "12",
+    designation: "Co-Founder",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/pqrst",
+    linkedin: "linkedin.com/in/pqrst",
+    phonenNo: "9898989898",
+    emailid: "pqrst@gmail.com"
+  },
+  {
+    name: "13",
+    designation: "President",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/abcdef",
+    linkedin: "linkedin.com/in/abcdef",
+    phonenNo: "9898989898",
+    emailid: "abcdef@gmail.com"
+  },
+  {
+    name: "14",
+    designation: "Member",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/pqrst",
+    linkedin: "linkedin.com/in/pqrst",
+    phonenNo: "9898989898",
+    emailid: "pqrst@gmail.com"
+  }
+]
+
+export default function teamMembers() {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+  const [index, setIndex] = React.useState(0);
+
+  const handleOpen = (data) => {
+    setOpen(true);
+    if (data != undefined) {
+      setIndex(data);
+    }
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <Fragment>
```

Comment:
I didn't understand the use of a Fragment here. you can directly export the div

Reply:
To return multiple html elements at a time

Reply:
What i mean is the Fragment has only one child So, might as well not have the fragment. 
Just a suggestion, you can use <></> in place of <Fragment></Fragment>